bson/compatibility.go

```diff
@@ -0,0 +1,16 @@
+package bson
+
+// Current state of the JSON tag fallback option. 
+var useJSONTagFallback = false
+
+// SetJSONTagFallback enables or disables the JSON-tag fallback for structure tagging. When this is enabled, structures
+// without BSON tags on a field will fall-back to using the JSON tag (if present).
+func SetJSONTagFallback(state bool) {
```

Comment:
Hi, 
I am nitpicking here, but could you please document (readme.md) for ex that even if there are no json/bson tags in the field comment, the value of that comment field would still be used?

```
struct A{
B string `c_field`
}

var a A{ "value" } 
``` 
would be marshaled as `{"c_field": "value"}` 
This behaviour is not found in any other encoder that I know of, and hence could be quite confusing if used by chance.